html/semantics/forms/form-submission-0/form-submission-algorithm.html

```diff
@@ -18,16 +18,33 @@
   assert_equals(counter, 2);
 }, 'If constructing entry list flag of form is true, then return');
 
-let test10 = async_test('Cannot navigate (after constructing the entry list)');
-test10.step(() => {
+
+async_test(t => {
+  let form = populateForm('<input type=hidden name=n value=v0><input type=submit>');
+  let submitter = form.querySelector('[type=submit]');
+  let iframe = form.previousSibling;
+  form.onsubmit = () => {
+    let hidden = form.querySelector('[type=hidden]');
+    hidden.value = 'inner click()';
+    submitter.click();
+    hidden.value = 'v2';
+  };
+  submitter.click();
+  iframe.onload = t.step_func_done(() => {
+    assert_not_equals(iframe.contentWindow.location.search.indexOf('n=v2'), -1);
+  });
+}, "If form's firing submit event is true, then return");
+
+let test9 = async_test('Cannot navigate (after constructing the entry list)');
+test9.step(() => {
```

Comment:
Was there a reason to change this from test10 to test9? Seems to just make the diff a bit more complicated.

Reply:
'10' was the step number in 'form submission algorithm', and this test case tests that step.  The specification PR changed the step number :)

Updating the step number every time won't work well.  I changed this to async_test(t => {}, 'Cannot navigate (...)').